components/data_comps/desp/cime_config/namelist_definition_desp.xml

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0"?>
+
+<?xml-stylesheet type="text/xsl" href="http://www.cgd.ucar.edu/~cam/namelist/namelist_definition.xsl"?>
+
+<entry_id version="2.0">
+
+  <!-- Each namelist variable is defined in an <entry> element.  The
+     content of the element is the documentation of how the variable is
+     used.  Other elements  are:
+     id
+          The variable's name.  *** N.B. *** The name must be lower case.
+          The module convert all namelist variable names to lower case
+          since Fortran is case insensitive.
+     type
+          An abbreviation of the fortran declaration for the variable.
+	  Valid declarations are:
+             char, integer, logical, real
+	  Any of these types may be followed by a comma separated list of
+	  integers enclosed in parenthesis to indicate an array.
+	  The current namelist validation code only distinquishes between
+	  string and non-string types.
+     category
+          A category assigned for organizing the documentation.
+     group
+          The namelist group that the variable is declared in.
+     valid_values
+          This is an optional attribute that is mainly useful for variables
+          that have only a small number of allowed values.
+  -->
+
+  <!-- =========================== -->
+  <!--   group desp_inparm         -->
+  <!-- =========================== -->
+
+  <entry id="desp_mode" modify_via_xml="DESP_MODE">
+    <type>char</type>
+    <category>desp</category>
+    <group>desp_nml</group>
+    <valid_values>NOCHANGE,DATATEST</valid_values>
+    <desc>
+      The mode of operation for the DESP model.
+      NOCHANGE: report  status of model 'pause' but make no changes
+      DATATEST: make a roundoff change  to the restart files of the
+                components in the 'PAUSE_COMPONENT_LIST' XML variable.
+      Default: NOCHANGE
+    </desc>
+    <values>
+      <value>$DESP_MODE</value>
+    </values>
+  </entry>
+
+  <entry id="info_debug">
+    <type>integer</type>
+    <category>desp</category>
+    <group>desp_nml</group>
+    <desc>
+      Level of debug output, 0=minimum, 1=normal, 2=more, 3=too much (default: 0)
+    </desc>
+    <values>
+      <value>0</value>
+    </values>
+  </entry>
+
+  <entry id="restfilm">
+    <type>char</type>
+    <category>desp</category>
+    <group>desp_nml</group>
+    <desc>
+      Model restart filename for the external system (ESP) model data.  This is
+      optional.  If restfilm is undefined, the restart filename will be
+      read from the DESP restart pointer file (or files for multiple instances).
+    </desc>
+    <values>
+      <value>undefined</value>
```

Comment:
As for pylint, I think I learned my lesson which is only run pylint on yellowstone. I didn't realize how different a slightly newer version of pylint can be. We need to agree on a version with the DOE folks (if they can internally agree) and stick with that.